File: js/Accepted/GRAPH/P2428_Capital/P2428_Capital.js
/*
Author : Teerapat Phokhonwong
Problem: 2428 - Capital
Link: https://www.beecrowd.com.br/judge/en/problems/view/2428
Answer: Accepted
Submission: 12/18/22, 6:42:37 PM
Runtime: 0.365s
*/
// var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var input = require('fs').readFileSync(__dirname+'/in.txt', 'utf8');
var lines = input.split('\n');

const [a, b, c, d] = lines.shift().split(" ");
if (a / b == c / d || a / b == d / c || a / c == b / d || a / c == d / b || a / d == b / c || a / d == c / b) {
    console.log("S");
}
else{
    console.log("N");
}

File: Python/src/uri_beecrowd/GRAPH/P2428_Capital.py
# author : Teerapat Phokhonwong
# Problem: 2428 - Capital
# Link: https://www.beecrowd.com.br/judge/en/problems/view/2428
# Answer: Accepted
# Submission: 12/18/22, 11:01:11 AM
# Runtime: 0.276s

a, b, c, d = map(int, input().split())

if a / b == c / d \
or a / b == d / c \
or a / c == b / d \
or a / c == d / b \
or a / d == b / c \
or a / d == c / b:
    print('S')
else:
    print('N')
